source/_patterns/02-molecules/vue-widget/__tests__/integration/vue-cards.spec.js

```diff
@@ -0,0 +1,85 @@
+import { render, Simulate } from 'vue-testing-library';
+import card from '../../src/vue-cards/components/card.vue';
+import cards from '../../src/vue-cards/components/cards.vue';
+import banner from '../../src/vue-cards/components/banner.vue';
+
+describe('card.vue', () => {
+  it('renders component with supplied props', () => {
+    const props = {
+      id: 77,
+      name: 'waffles',
+      phone: '555-555-5555',
+      website: 'waffles.com',
+      email: 'admin@waffles.com',
+    };
+
+    const { getByText } = render(card, { props });
+    expect(getByText('waffles').textContent).toContain('waffles');
+    expect(getByText('555-555-5555').textContent).toBe('555-555-5555');
+    expect(getByText('waffles.com').textContent).toBe('waffles.com');
+    expect(getByText('admin@waffles.com').textContent).toContain('admin');
+  });
+
+  it('initializes isClicked status as false', () => {
+    const { getByText } = render(card);
+    const clickStatus = getByText('clicked:', { exact: false });
+
+    expect(clickStatus.textContent).toBe('clicked: false');
+  });
+});
+
+describe('banner.vue', () => {
+  it('shows the username passed in', () => {
+    const props = {
+      username: 'krieger',
+    };
+
+    const { getByText } = render(banner, { props });
+    expect(getByText('krieger', { exact: false }).textContent).toContain(
+      'krieger'
+    );
+  });
+
+  it('changes color on click', () => {
+    const bannerElement = document.querySelector('.vue-banner');
```

Comment:
Curious why this test doesn't use the vue-testing-library. Also, [jest-dom](https://github.com/gnapse/jest-dom) can import some nice selectors that may also be good to include in the project. Larger point of conversation that may expand past this PR.

Reply:
I don't think I knew how to target classes with it, and I think at the time I was trying to avoid adding testid. I can change it to testid though

Reply:
As for babel, is there a way to import it like I'm doing in the jest config? I know it's a bit redundant, but I wasn't sure if there was another way

Reply:
Oh, I see that we already had @babel/polyfill now. D'oh!

Reply:
Yep. We just need to remove the v6 version of the polyfill from the dependencies and import the babel namespaced import that is for v7.

Reply:
Benefit I could see of using testids is that it also maintains a consistent testing approach. There's not anything wrong with the approach, just my thoughts as I have been heavily steeped in Jest testing lately.

Reply:
So, troubleshooting some odd behavior here - testid doesn't seem to be working on just this test: 

```
banner.vue › changes color on click

    Unable to find an element by: [data-testid="vue-banner"]

    <div
      class="vue-banner"
      data-testid="vue-banner"
      style="background: rgb(0, 0, 0);"
    >
      <h3>
        <marquee>
          <span>
            krieger was clicked!
          </span>
        </marquee>
      </h3>
    </div>

```

Reply:
I'm rolling this back to the previous version for now